app/marketing/migrations/0023_auto_20180507_1646.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.0.4 on 2018-05-07 16:46
+
+from django.db import migrations
+
+
+def insert_alumni(apps, schema_editor):
+    Profile = apps.get_model('dashboard', 'Profile')
+    Alumni = apps.get_model('marketing', 'Alumni')
+    elements = [
+        ('PixelantDesign', 'gitcoinco'),
+        ('mbeacom', 'gitcoinco'),
+        ('StareIntoTheBeard', 'gitcoinco'),
+        ('thelostone-mc', 'gitcoinco'),
+        ('KamesCG', 'uport'),
+        ('SaptakS', 'gitcoinco'), # TODO: Do Gitcoinco alumni count?
+        ('eswarasai', 'MARKETProtocol'),
+        ('Perfectmak', 'MARKETProtocol'),
+        ('lazaridiscom', 'Metamask'),
+        ('voith', 'ethereum'),
+        ('dylanjw', 'ethereum'),
+    ]
+    for ele in elements:
+        Alumni.objects.create(
+            profile=Profile.objects.get(handle__iexact=ele[0]),
```

Comment:
This migration bombs on local since these `Profile` objects don't exist.  I'm thinking this might be a good opportunity for us to introduce our first round of fixtures.  🤔